src/api/app/controllers/status/required_checks_controller.rb

```diff
@@ -76,17 +79,15 @@ def set_project
 
   def set_checkable
```

Comment:
comparing both methods, I cannot see why your version is better with ```@checkable``` being mutated and nested if blocks.. any opinion on that @openSUSE/open-build-service ?

Reply:
this is not about 'better', this is adding a branch

Reply:
sure, but wasn't possible to do it keeping the method similar with the old one or breaking it in two methods each one checking for a parameter? IMO the new version looks more complex than before, what do you think?

Reply:
Feel free to refactor within the status api sprint - whenever it starts.